README.md

```diff
@@ -12,7 +13,57 @@ Main Tensorflow Training Repository for Subbots
 
 Clone repository with 'git clone --recursive https://github.com/ubc-subbots/Object-Detection-Training'
 
-To install the dependencies of the repository, run the ./setup_environment.sh script. Anytime you want to use the tensorflow api, run 'source subbots_python/bin/activate'.
+To install the dependencies of the repository, run the ./setup_environment.sh script. Anytime you want to use the
+tensorflow api, run 'source subbots_python/bin/activate'.
```

Comment:
In general, when writing markdown files, we should leave all the text on one line. This lets the browser wrap the text as needed when it renders the markdown doc, rather then us manually doing it, in turn making it more readable.

Reply:
Sounds good I'll fix this